beacon-chain/p2p/service.go

```diff
@@ -226,6 +226,22 @@ func (s *Service) Start() {
 
 	// Periodic functions.
 	async.RunEvery(s.ctx, params.BeaconNetworkConfig().TtfbTimeout, func() {
+		// every time reset peersToWatch, add RelayNodes and trust peers
```

Comment:
maybe a better thing to do here would be to add this whole chunk into the actual method `ensurePeerConnections` . Splitting it this way seems an unclean approach, since `ensurePeerConnections` is executed every `ttfb_timeout`